GVFS/GVFS.Common/Maintenance/GitMaintenanceStep.cs

```diff
@@ -79,18 +79,28 @@ public void Stop()
 
                 if (process != null)
                 {
-                    if (process.TryKillRunningProcess())
+                    if (process.TryKillRunningProcess(out string processName, out int exitCode, out string error))
                     {
                         this.Context.Tracer.RelatedEvent(
                             EventLevel.Informational,
-                            this.Area + ": killed background Git process during " + nameof(this.Stop),
+                            string.Format(
+                                "{0}: killed background process {1} during {2}",
+                                this.Area,
+                                processName,
+                                nameof(this.Stop)),
                             metadata: null);
                     }
                     else
                     {
                         this.Context.Tracer.RelatedEvent(
                             EventLevel.Informational,

```

Comment:
Should this be a warning level?

Reply:
It wasn't before, and I don't think we need it necessarily in telemetry.

Reply:
Do warnings get reported to telemetry by default? I thought that was only `RelatedError`...

I think `Warning` would be appropriate for events that might be unexpected and could indicate a problem.

Reply:
> Do warnings get reported to telemetry by default? I thought that was only `RelatedError`

That's correct, warnings will not be reported to telemetry by default